cmd/gateway/commands.go

```diff
@@ -291,4 +297,42 @@ func createProvisionerModeCommand() *cobra.Command {
 			})
 		},
 	}
+
+	cmd.PersistentFlags().Var(
+		&gatewayCtlrName,
+		gatewayCtrlNameFlag,
+		fmt.Sprintf(gatewayCtrlNameUsageFmt, domain),
+	)
+	utilruntime.Must(cmd.MarkPersistentFlagRequired(gatewayCtrlNameFlag))
+
+	cmd.PersistentFlags().Var(
+		&gatewayClassName,
+		gatewayClassFlag,
+		gatewayClassNameUsage,
+	)
+	utilruntime.Must(cmd.MarkPersistentFlagRequired(gatewayClassFlag))
+
+	return cmd
+}
+
+func createSleepCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "sleep",
+		Short: "Sleeps for specified duration and exits",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			logger := ctlrZap.New()
+			logger.Info(
+				"Executing sleep command",
+				"version", version,
+				"commit", commit,
+				"date", date,
+			)
+
+			duration := 30 * time.Second
```

Comment:
Is there a way to make this configurable by the user?

Reply:
What do you think of taking just one required argument in seconds?

For example:
```
/gateway sleep 30
```
will sleep for 30s

Reply:
Yeah I think that would work!